server/src/main/java/com/exacaster/lighter/application/sessions/SessionHandler.java

```diff
@@ -49,10 +52,37 @@ public void launch(Application application, Consumer<Throwable> errorHandler) {
         app.launch(backend.getSubmitConfiguration(application));
     }
 
+    @EventListener
+    @Async
+    public void startPermanentSession(StartupEvent event) {
+        sessionService.fetchPermanent().ifPresent(it -> sessionService.deleteOne(it.getId()));
```

Comment:
I think it would be simpler to provide permanent session id in session configuration, in that case, different session type will not be needed.

Reply:
Good idea regarding permanent session id from configuration file, but separate type is needed anyway

Reply:
What for?

Reply:
Permanent session lifecycle is different from regular session.